lib/protobuf/field/field_array.rb

```diff
@@ -65,7 +65,7 @@ def normalize(value)
         fail TypeError, "Unacceptable value #{value} for field #{field.name} of type #{field.type_class}" unless field.acceptable?(value)
 
         if field.is_a?(::Protobuf::Field::EnumField)
-          field.type_class.fetch(value)
+          fetch_enum(field.type_class, value)
```

Comment:
These seems to be the only code that actually introduces behavior change to the existing interface, so I want to track this pretty closely. I'll play with this to see how a `nil` value behaved here before.